components/Common/BlogPostCard/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { useMemo } from 'react';
+import type { ComponentProps, FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Preview from '@/components/Common/Preview';
+
+import styles from './index.module.css';
+
+const dateTimeFormat = new Intl.DateTimeFormat(navigator.language, {
+  day: 'numeric',
+  month: 'short',
+  year: 'numeric',
+});
+
+type Author = {
+  fullName: string;
+  src: ComponentProps<typeof AvatarGroup>['avatars'][number]['src'];
+};
+
+export type BlogPostCardProps = {
+  title: ComponentProps<typeof Preview>['title'];
+  type: Required<ComponentProps<typeof Preview>>['type'];
+  description: string;
+  authors: Author[];
+  date: Date;
+};
+
+const BlogPostCard: FC<BlogPostCardProps> = ({
+  title,
+  type,
+  description,
+  authors,
+  date,
+}) => {
+  const avatars = useMemo(() => {
+    const parsedAvatars = authors.map(({ fullName, src }) => ({
+      alt: fullName,
+      src,
+      toString: () => fullName,
+    }));
+
+    return parsedAvatars;
+  }, [authors]);
+
+  const formattedDate = useMemo(
+    () => ({
+      ISOString: date.toISOString(),
+      short: dateTimeFormat.format(date),
+    }),
+    [date]
+  );
+
+  return (
+    <article className={styles.container}>
+      <Preview
+        title={title}
+        type={type}
+        height="auto"
+        className={styles.preview}
+      />
+      <p className={styles.subtitle}>
+        <FormattedMessage id={`components.common.card.${type}`} />
+      </p>
+      {/* We hide it from screen-readers as Preview component already wraps the title in an h2 tag */}
```

Comment:
I'm often opposed to inline comments within JSX. I would say these things are self-explanatory.

Reply:
I did not know if it was going to be self-explanatory, although it's true that there's a unit test covering this, so I will remove the comment.